graphql/schemas/Social/messagesTypes.graphql

```diff
@@ -0,0 +1,36 @@
+
+input CreateMessageTypeInput {
+    languages_id: Int!
+    name: String!
+    verb: String!
+    template: String!
+    templates_plura: String!
+}
+
+type MessageType {
+    id: Int!
+    languages_id: Int!
+    apps_id: Int!
+    uuid: String!
+    name: String!
+    verb: String!
+    template: String!
+    templates_plura: String!
```

Comment:
@FredPeal missing a l

Reply:
![image](https://github.com/bakaphp/kanvas-ecosystem-api/assets/22231077/cfef0585-3398-4f12-99a8-762cf4cfc590)
You create the field without a l